src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/ShowDropColumnsBase.cs

```diff
@@ -155,6 +164,11 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
             }
 
             returnType = colsToKeep;
+            if (newDisplayNameMapping != null)
```

Comment:
discussed, potential breaking change if an outerscope had the same set of display names, this could change which scope a name gets resolved from